src/components/App.js

```diff
@@ -84,7 +84,7 @@ export class App extends Component {
         <Route exact path='/login' render={ (props)=> <Login {...props}/>} />
         <Route path='/' render={ () => <Nav /> } />
         <Route exact path='/' render={() => <Main toggleFavorites={this.toggleFavorites} /> } />
-        <Route exact path='/favorites' render={ () => <MovieList toggleFavorites={this.toggleFavorites}movies={this.props.favorites} /> } />
+        <Route exact path='/favorites' render={ () => <Favorites toggleFavorites={this.toggleFavorites} movies={this.props.favorites} /> } />
```

Comment:
I think we do need a separate container here since the logic is a bit more complex.